csharp/Urban.DCP.Handlers/LoginHandler.cs

```diff
@@ -33,6 +33,7 @@ protected override void InternalGET(HttpContext context, HandlerTimedCache cache
                                     user.Name,
                                     Admin = user.IsSysAdmin(),
                                     Limited = user.IsLimited(),
+                                    IsNetworked = user.IsNetworked(),
```

Comment:
Minor, but for parity with the other returned properties, consider renaming this to `Networked`.